NitroxModel/Serialization/ServerList.cs

```diff
@@ -84,13 +87,25 @@ public void RemoveAt(int index)
             entries.RemoveAt(index);
         }
 
+        public void CleanLanServers()
+        {
+            for (int i = entries.Count - 1; i >= 0; i--)
+            {
+                if (entries[i].IsLan)
+                {
+                    entries.RemoveAt(i);
+                }
+            }
+        }
+
         public class Entry
         {
             public string Name { get; }
             public string Address { get; }
             public int Port { get; }
+            public bool IsLan { get; }
```

Comment:
`IsLan` is currently only used for changing persistence so let's rename it to match. Note to invert the bool code using IsLan.

```suggestion
            /// <summary>
            ///     If true, entry will be saved to storage.
            /// </summary>
            public bool Persist { get; }
```